tests/regression/test_barrier_before_return.cpp

```diff
@@ -1,166 +1,166 @@
-/* Tests a kernel with a barrier as the last statement.

```

Comment:
I prefer not to push mass white space cleanup commits since they make 'git blame' more difficult to track etc.

Reply:
OK, dropped